torch/_export/serde/schema.py

```diff
@@ -185,6 +185,13 @@ class NamedArgument:
     arg: Argument
 
 
+@dataclass
+class NNModuleFrame:
```

Comment:
where is this used?

Reply:
It is used during `serialize_metadata()` and `deserialize_metadata()` to schematize the `nn_module_stack` field

Reply:
@yiming0416 I guess to really "use" it you need to put it into some fields and make it part of the type definition.